lib/core/settings/src/types/settings.dart

```diff
@@ -554,6 +567,121 @@ class ImageViewerSettings extends Equatable {
   }
 }
 
+class TallMediaSettings extends Equatable {
+  const TallMediaSettings({
+    required this.aspectRatioThreshold,
+    required this.minHeightPx,
+    required this.minViewportHeightRatio,
+    required this.minPixelCount,
+    required this.navigationVelocityThreshold,
+    required this.navigationDistanceThreshold,
+    required this.edgeActivationRatio,
+    required this.scrollLockVelocityThreshold,
+    required this.scrollLockDistanceThreshold,
+    required this.minScrollExtentPx,
+  });
+
+  const TallMediaSettings.defaults()
```

Comment:
This is too much for an user settings. Should have just create a new value in PostDetailsSwipeMode, maybe comic or webtoon.